docs/Data_Portal/Users_Guide/cohort_builder.md

```diff
@@ -90,3 +90,41 @@ Users can then perform the following actions:
 
 * Download files associated with the cohort from the [Repository](Repository.md)
 * Analyze data from the cohort in the [Analysis Center](analysis_center.md)
+
+
+## Cohort Types
+
+Depending on how they are modified or created, cohorts can have different types of filters and thus behave differently:
+* __General Queries__
+* __Specific List of Cases__
+
+### General Queries
```

Comment:
not loving calling them 'general queries'. any thoughts, @wwysoc2 ?

Reply:
Maybe "Custom Queries"? I think it gives the sense that users are always in control here, which they are.

Reply:
Also, add a return before the bullets.

Reply:
all done. lemme know if it's ok to merge. @wwysoc2